gradle.properties

```diff
@@ -4,5 +4,4 @@ version=1.7.14-SNAPSHOT
 buildDir=buildGradle
 mavenSnapshotRepo=https://oss.sonatype.org/content/repositories/snapshots
 mavenReleaseRepo=https://oss.sonatype.org/service/local/staging/deploy/maven2/
-org.gradle.caching=true
```

Comment:
Is this an intentional change? Because it means for example that when running `./gradlew jacocoReport`, it wont do that on the cached results of the last testrun, but instead needs to run all the tests again (I think).

If this is intentional and will stay this way, the docs for updating the test262.properties file need a slight adjustment, because it now states you need to run with the `--rerun-tasks` param, because the `-DupdateTest262properties` option doesn't work based on cached results. So, if we're not doing caching anymore, the `--rerun-tasks` param is no longer needed

Reply:
I'll take this out of that particular PR. But I have been finding that the SpotBugs plugins are failing randomly on my machine and it seemed to be caching related -- I'd otherwise like to leave it on.